qbft/spectest/tests/proposal/invalid_value_check.go

```diff
@@ -1,32 +1,27 @@
 package proposal
 
 import (
+	"github.com/bloxapp/ssv-spec/qbft"
 	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
 )
 
 // InvalidValueCheck tests a proposal that doesn't pass value check
 func InvalidValueCheck() *tests.MsgProcessingSpecTest {
-	panic("implement")
+	pre := testingutils.BaseInstance()
```

Comment:
if this test fails, what will the failure/error look like?

Reply:
```
        	Error:      	An error is expected but got nil.
```